_includes/commands/advtable-cmds.md

```diff
@@ -15,6 +16,11 @@
 - `sort` - `'row'`, `'column'`, `'selection'`, or `'table'`
 - `order` - `'ascending'` or `'descending'`
 
+`mceTableToggleSeries` accepts an object with the following key-value pairs:
+
+- `name` - a string indicating the series to toggle. Series and their associated names are configured using the [advtable_value_series]({{site.baseurl}}/plugins/premium/advtable/#advtable_value_series) option. If a different series name is provided than what is present on the table, the series column will be updated to use the new series.
+- `location` - `'left'`
```

Comment:
I feel like we should put a sentence here explaining that `'left'` isn't an example of one of the values you can put here, it's the only value you can put here. Then maybe say some words about the potential for forwards compatibility, if we even have anything to say there?